config/categories.yml

```diff
@@ -1,25 +1,49 @@
 ---
-- label : _('Hotel')
```

Comment:
Isn't it a mistake here to replace `_('Hotel')` by `hotel` ?

Reply:
No, there is already an existing key named `hotel` in the po, so that's why trads won't work in the branch 709 for the categories